plugins/postcss-progressive-custom-properties/src/match.ts

```diff
@@ -12,12 +12,23 @@ export function matches(a, b) {
 	}
 
 	if (a.nodes && b.nodes) {
-		if (a.nodes.length !== b.nodes.length) {
-			return false;
-		}
-
 		for (let i = 0; i < a.nodes.length; i++) {
-			if (!matches(a.nodes[i], b.nodes[i])) {
+			let ia = i;
+			let ib = i;
+
+			while (a.nodes[ia] && a.nodes[ia].type === 'space') {
+				ia++;
+			}
+
+			while (b.nodes[ib] && b.nodes[ib].type === 'space') {
+				ib++;
+			}
```

Comment:
`space` nodes are now ignored when matching.
Makes it easier to match cases with irregular spacings : `rgba(255,0, 255 , 1)`